Sources/SimpleSalesforceSDK/SSSDK.swift

```diff
@@ -99,9 +99,16 @@ public class SSSDK {
     return self.auth.isAuthenticated
   }
 
-  /// Erases all tokens and expiry date from keychain
+  /// Revokes the access token from salesforce and erases all tokens and expiry date from keychain
   public func logout() {
-    self.auth.reset()
+    self.auth.revokeAccessToken(config: config!) { error in
```

Comment:
since this is now async, shouldn't the logout method also have a completion handler now?